CONTRIBUTION.md

```diff
@@ -64,3 +64,16 @@ Bug: T42
 The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
 lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
 git tools.
+
+### PR workflow
+
+- Before submitting a PR
+  - Format your code and fix all the spelling/grammatical mistakes.
+  - Limit the column length to 120 characters.
+  - Error messages should not be more than 2 lines long.
+- Following are the things you should keep in mind once you open a PR:
+  - Wait for comments from at least 2 reviewers before pushing changes. Open comments in the meanwhile can be worked on locally.
+  - PR comments should be addressed in new commits. Before final approval, they have to be squashed.
+  - Maintain changes to multiple commits. (see below)
+
+![Sample PR life-cycle](/images/PR-workflow.jpg)
```

Comment:
Perhaps some text explaining the diagram would help.

Reply:
The diagram on its own will be hard to interpret. I would let @gmarkley-VI come up with the diagram and you can add it in a follow up PR. For now you could a few sentences explaining what maintaning changes to multiple commits mean. How about `If your PR has multiple commits, changes requested should eventually be squashed into the original commit where the change was requested before the final merge.`?

Reply:
Sounds good. I'll try to use this, or come up with a couple of sentences to explain well in the next commit.